libs/sdk-ui-tests/report.20200922.120250.11081.0.001.json

```diff
@@ -0,0 +1,815 @@
+{
```

Comment:
I would say that this file should not be commited

Reply:
Removed